### PR TITLE
First custom styles

### PR DESCRIPTION
**Pull request by going-confetti**

### CODE REVIEW

File: src/app.pcss [hunk 1]
```diff
@@ -53,7 +54,7 @@ body {
     --radius: 0.5rem;
   }
 
-  .dark {
+  /* .dark {
```
Comment (going-confetti) [inline src/app.pcss:57]: Couldn't find a better way to force the light theme 🤔 🤦‍♂️

File: src/app.pcss [hunk 2]
```diff
@@ -36,7 +36,8 @@ body {
     --border: 214.3 31.8% 91.4%;
     --input: 214.3 31.8% 91.4%;
 
-    --primary: 250 93.7% 68.6%;
+    /* --primary: 250 93.7% 68.6%; */
+    --primary: #7d64ff;
```
Comment (going-confetti) [inline src/app.pcss:40]: This didn't work like I assume it would. Do I need to provide HSL values?

Comment (Llandy3d) [reply src/app.pcss:40]: I'm not knowledgeable in this part but worth trying with the same value system as before

**Review by Llandy3d — APPROVED** [review]
✨

File: src/app.pcss (hunk 1 above)
Comment (allansson) [reply src/app.pcss:57]: I used `setMode` from `mode-watcher` somewhere in the code 🤔

**Pull request merged**